src/ergw_aaa_diameter.erl

```diff
@@ -345,6 +345,13 @@ format_cc(Value) when is_binary(Value) ->
                              || <<X>> <= Value]);
 format_cc(_) -> [].
 
+octet_string([Int]) when is_integer(Int) -> [<<Int>>];
+octet_string([{A, B}]) when is_integer(A), is_integer(B) -> [<<A, B>>];
```

Comment:
I don't like this type of special handling. Constructs like this are the reason why I usually code the value to request mapping as fold over the session with a special handler per attribute. That handler can then take care of the encoding rules for the attribute.